src/module.c

```diff
@@ -2165,7 +2181,14 @@ void RM_Yield(RedisModuleCtx *ctx, int flags, const char *busy_reply) {
  * REDISMODULE_OPTIONS_HANDLE_REPL_ASYNC_LOAD:
  * Setting this flag indicates module awareness of diskless async replication (repl-diskless-load=swapdb)
  * and that redis could be serving reads during replication instead of blocking with LOADING status.
- */
+ *
+ * REDISMODULE_OPTIONS_ALLOW_NESTED_KEYSPACE_NOTIFICATIONS:
+ * Declare that the module want to get nested key space notifications.
+ * By default, Redis will not fire key space notification that happened inside
+ * a key space notification callback. This flag allows to change this behavior
+ * and fire nested key space notifications. Notice, if enabled, the module
+ * should protected himself from endless loops that might happened due to recursive
+ * key space notifications. */
```

Comment:
```suggestion
 * Declare that the module wants to get nested key-space notifications.
 * By default, Redis will not fire key-space notifications that happened inside
 * a key-space notification callback. This flag allows to change this behavior
 * and fire nested key-space notifications. Notice: if enabled, the module
 * should protected itself from infinte recursion. */
```

Fixed a typo and went on a bit...